rustls/tests/api.rs

```diff
@@ -4632,6 +4632,14 @@ fn early_data_is_available_on_resumption() {
         .unwrap()
         .exporter()
         .unwrap();
+    assert_eq!(
+        Some(Error::ApiMisuse(ApiMisuse::ExporterAlreadyUsed)),
```

Comment:
Nit: I guess there's quite a bit of this in the existing code, but personally I think observed to be on left and expected on the right is more idiomatic. (I forget if we've had this discussion before?)